modules/core/src/main/java/org/apache/ignite/internal/processors/platform/client/cache/ClientCacheIndexQueryRequest.java

```diff
@@ -102,6 +107,9 @@ public ClientCacheIndexQueryRequest(
 
         if (limit > 0)
             qry.setLimit(limit);
+
+        if (label != null)
+            qry.setLabel(label);
```

Comment:
why we can`t just set null label ?

Reply:
It seemed to me that this way the code looks more consistent.
But I changed it as you wanted.